Classes/JRHostsFile.h

```diff
@@ -1,5 +1,11 @@
 @interface JRHostsFile : NSObject
 
 -(id)init;
+-(id)initWithFileName:(NSString*)fileName;
+
+-(NSString*)defaultFileName;
```

Comment:
The default file name never changes, so it should be a constant. I think by convention they are Capitalized.

```objc
NSString *const DefaultFileName = @"/etc/hosts";
```